docs/azure-saas-docs/content/en/quick-start.md

```diff
@@ -0,0 +1,76 @@
+- Depending on the project you wish to run, you'll need to set some secrets to properly setup authentication with B2C. See the [App Settings](#app-settings) section below
+
+### App Settings
+When running the project locally, you will need to set some app settings & secrets manually using the [.NET secrets manager](https://docs.microsoft.com/en-us/aspnet/core/security/key-vault-configuration?view=aspnetcore-6.0#secret-storage-in-the-development-environment). When deployed to azure (below), these secrets are automatically configured for you and stored in Azure Key Vault.
+
+Make sure you check out the [readme files](#more-info) in each project's directory for a description of the app settings & secrets you'll need to set in order to run the respective project.
+
+
+## 2.b. Deploying to Azure - Entire Solution
+
+Deploying to Azure is easy thanks to our pre-configured ARM (Azure Resource Manager) templates.
+
+This button will take you to the Azure portal, passing it the template. You'll be asked a few questions, and then the solution will be up and running in just a few minutes. You will need your B2C configuration values and secrets from step 1.
+
+**Deployment Coming Soon!**
+<!-- [![Deploy to Azure](https://www.azuresaas.net/assets/images/deploy-to-azure.svg)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-saas%2Fmain%2Fsrc%2FSaas.Deployment%2FSaas.Deployment.Root%2Fazuredeploy.json/createUIDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-saas%2Fmain%2Fsrc%2FSaas.Deployment%2FSaas.Deployment.Root%2FcreateUiDefinition.json) -->
+
+## 2.c. (Advanced) Deploying to Azure - Single Module
+
+If you'd like to use just one (or more) module from the project, we've provided bicep templates to do that as well. In each project directory, you'll find a folder named `{ModuleName}.Deployment` that contains all the bicep code you'll need to deploy just that Module. Please be advised that there are certain dependencies that each module requires in order for it to deploy properly. You may find that you need to edit the bicep templates to match your use case. You will find instructions and a list of dependencies for each module within the [module's readme](#more-info).
+
+## 3. (Optional) Configure Email Provider
+
+If you choose to deploy the entire solution to Azure, one of the things that gets deployed is the SaaS.Notifications module **need page and link**, an Azure Logic App. This logic app has a simple HTTP trigger that the SaaS.SignupAdministration site is configured to make a POST request to in order to send emails (ie, at the end of the tenant creation process). By default, there is no email provider configured and the emails do not actually get sent. If you'd like to enable email sending, you will need to manually configure your email provider connector of choice inside the Logic App. See the instructions [here](components/saas-notifications.md) to get started.
```

Comment:
```suggestion
The SaaS.Notifications module **need page and link** is an Azure Logic App responsible for generating email notifications. By default, there is no email provider configured. If you'd like to enable email notifications, you will need to manually configure your email provider connector of choice inside the Logic App. See the instructions [here](components/saas-notifications.md) to get started.
```

Recommending we truncate this section and let those be disclosed in components/saas-notifications.md. Since this document is a quick start, details like this can wind up more as noise than signal.